bndtools.core/_plugin.xml

```diff
@@ -871,6 +871,23 @@
 				description="Disable verification of server's X509 certificate (insecure, for testing only!)" />
 		</plugin>
 
+		<plugin class="aQute.bnd.repository.maven.provider.MavenBndRepository"
```

Comment:
I had no idea this existed!

Reply:
You mean those sections in the `_plugin.xml` ?

Yes, that is how I came to this PR? I was trying the +-Button of the Plugins list, because I wanted to see what this is... because I have never seen anything in the Plugins list  of my build.bnd. And tadaaa I saw that I could add repositories. Unfortunately I did not see the MavenRepo we are using. 
 
We can later add also support for other repositories if needed. The xml I created from `aQute.bnd.repository.maven.provider.Configuration`. I guess  we can do the same for `aQute.bnd.repository.maven.pom.provider.PomConfiguration`.